cfgov/unprocessed/js/routes/common.js

```diff
@@ -8,6 +8,7 @@
 require( 'jquery' );
 require( 'jquery-easing' );
 require( 'cf-expandables' );
+require( '../modules/util/modernizr');
```

Comment:
Used to remove styles for browsers that don't support pointer-events here: https://github.com/cfpb/cfgov-refresh/pull/1229/files#diff-584caab787ffa39e459a947a816ba2e4R83

Rational here: https://github.com/cfpb/cfgov-refresh/pull/1229#discussion_r47845271